packages/admin/admin-icons/src/CometColor.tsx

```diff
@@ -1,23 +1,33 @@
 import { SvgIcon, SvgIconProps } from "@mui/material";
+import useConstant from "use-constant";
+import { v4 as uuidv4 } from "uuid";
+
+function getUniqueId() {
+    return `comet-icon-${uuidv4()}`;
+}
 
 export default function Comet(props: SvgIconProps) {
+    const uniqueIdA = useConstant(() => getUniqueId());
+    const uniqueIdB = useConstant(() => getUniqueId());
+    const uniqueIdC = useConstant(() => getUniqueId());
```

Comment:
```suggestion
    const uniqueIdA = useConstant(getUniqueId);
    const uniqueIdB = useConstant(getUniqueId);
    const uniqueIdC = useConstant(getUniqueId);
```

No need for additional functions